### ISSUE TITLE
Consider documenting some deprecations 

### ISSUE THREAD

**Issue by stronk7** [sebastianbergmann/phpunit-documentation-english#217]
Was looking for the impact of the move from PHPUnit 8 to PHPUnit 9, when I saw these 2 deprecations/removals (and it seems that there were way more, as listed @ sebastianbergmann/phpunit#4210):

- assertRegExp() (#4086)
- assertEqualXMLStructure() (#4091)

That, apparently found their route in PHPUnit 9.1, not 9.0.

This issue is about consider when all those "extra" (not present in 9.0) changes should be commented @ https://phpunit.de/announcements/phpunit-9.html or other useful places, like the Changelog (maybe keeping it for the whole life of a major version - 9.x - wouldn't be crazy).

tl;dr; right now it's impossible to know in advance all the things that have been deprecated / removed since 8.x to 9.x. Just that.

Ciao :-)

**Comment by sebastianbergmann** [comment]
I agree that this could be improved but the information, while fragmented, is there:

* https://github.com/sebastianbergmann/phpunit/blob/8.0.0/ChangeLog-8.0.md#800---2019-02-01
* https://github.com/sebastianbergmann/phpunit/blob/8.1.0/ChangeLog-8.1.md#810---2019-04-05
* https://github.com/sebastianbergmann/phpunit/blob/8.2.0/ChangeLog-8.2.md#820---2019-06-07
* https://github.com/sebastianbergmann/phpunit/blob/8.3.0/ChangeLog-8.3.md#830---2019-08-02
* https://github.com/sebastianbergmann/phpunit/blob/8.4.0/ChangeLog-8.4.md#840---2019-10-04
* https://github.com/sebastianbergmann/phpunit/blob/8.5.0/ChangeLog-8.5.md#850---2019-12-06
* https://thephp.cc/news/2020/02/migrating-to-phpunit-9

**Comment by stronk7** [comment]
Yeah, I've created here a similar list with the PHPUnit 9 sources (that is the one I'm interested now, already using 8.5 here) that we are gong to need.

I didn't mention the cool [Migrating to PHPUnit 9](https://thephp.cc/news/2020/02/migrating-to-phpunit-9) because I assumed it's sort of blog/news and didn't have much sense to go adding info there. Although it's the 1st place I always visit, maybe keeping the list of meaningful links to changelogs there makes sense, don't know.

In this case it's specially "unfortunate" (well, it's not the end of the world), because many of the changes happened with 9.1, so they aren't in the original 9.0 release/announcement/migration docs.

Anyway, here it's my list (after having looked to all 9.0 to 9.5 changelogs) and I'll be using them as references for our migration:

- https://phpunit.de/announcements/phpunit-9.html
- https://thephp.cc/news/2020/02/migrating-to-phpunit-9
- https://github.com/sebastianbergmann/phpunit/blob/9.0.0/ChangeLog-9.0.md
- https://github.com/sebastianbergmann/phpunit/blob/9.1.0/ChangeLog-9.1.md

Thanks for the feedback! Ciao :-)

**Comment by sebastianbergmann** [comment]
If it's just a list of links to ChangeLogs you're looking for then look no further than https://phpunit.de/announcements/.

**Comment by stronk7** [comment]
Good point, it's enough.
